nemo/collections/asr/parts/submodules/ctc_decoding.py

```diff
@@ -580,6 +585,13 @@ def compute_confidence(self, hypotheses_list: List[Hypothesis]) -> List[Hypothes
                 hyp.word_confidence = self._aggregate_token_confidence(hyp)
         return hypotheses_list
 
+    @abstractmethod
+    def get_words_offsets(self) -> List[Dict[str, Union[str, float]]]:
```

Comment:
Please, unify the signature of the `get_words_offsets` for abstract method and implementation